### PR TITLE
Add namespace for kubectl exec cmd

### PR DESCRIPTION
**Pull request by fisherxu**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/mpi-operator/101)

### CODE REVIEW

File: pkg/controllers/mpi_job_controller.go [hunk 1]
```diff
@@ -854,8 +854,8 @@ func newConfigMap(mpiJob *kubeflow.MPIJob, workerReplicas int, processingUnitsPe
 set -x
 POD_NAME=$1
 shift
-%s/kubectl exec ${POD_NAME} -- /bin/sh -c "$*"
-`, kubectlMountPath)
+%s/kubectl exec ${POD_NAME} -- /bin/sh -n%s -c "$*"
```
Comment (rongou) [inline pkg/controllers/mpi_job_controller.go]: Are you sure this works? You are passing a `-n<namespace>` to `/bin/sh`, which doesn't really make sense.

Comment (fisherxu) [reply pkg/controllers/mpi_job_controller.go]: Sorry for the delay, have addressed :) If we don't deloy the mpijob in default ns, this options is necessary.

Comment (fisherxu) [reply pkg/controllers/mpi_job_controller.go]: In my environment, don't use the serviceaccount to connect to kube-apiserver, instead set the kubeconfig by myself. So it's not necessary in the general scenario.

**Pull request closed without merging**